src/components/input-field/input-field.stories.js

```diff
@@ -1,81 +1,85 @@
-// import { html } from "lit-html";
-// import "../input-field";
-// import styles from "./styles.scss";
+import { html } from "lit-html";
+import "../input-field";
+import styles from "./styles.scss";
+import { stylesheet, docsIntro } from "../../../.storybook/utils.js";
 
-// const defaultArgs = {
-//   block: false,
-//   disabled: false,
-//   error: false,
-//   small: false,
-//   success: false,
-// };
+const defaultArgs = {
+  block: false,
+  disabled: false,
+  error: false,
+  small: false,
+  success: false,
+};
 
-// export default {
-//   title: "native-elements/vl-input-field",
-//   args: { ...defaultArgs },
-//   argTypes: {
-//     block: {
-//       name: "data-vl-block",
-//       type: { summary: "boolean" },
-//       description: "Het input-veld zal de breedte van zijn parent aannemen",
-//       table: {
-//         defaultValue: { summary: "false" },
-//       },
-//     },
-//     disabled: {
-//       name: "data-vl-disabled",
-//       type: { summary: "boolean" },
-//       description: "Schakelt interactie door de gebruiker uit",
-//       table: {
-//         defaultValue: { summary: "false" },
-//       },
-//     },
-//     error: {
-//       name: "data-vl-error",
-//       type: { summary: "boolean" },
-//       description:
-//         "Zorgt ervoor da er een rode rand rond het input-veld verschijnt",
-//       table: {
-//         defaultValue: { summary: "" },
-//       },
-//     },
-//     small: {
-//       name: "data-vl-small",
-//       type: { summary: "boolean" },
-//       description: "Kleine variant van het input-field",
-//       table: {
-//         defaultValue: { summary: "false" },
-//       },
-//     },
-//     success: {
-//       name: "data-vl-success",
-//       type: { summary: "string" },
-//       description:
-//         "Zorgt ervoor dat er een groene rand rond het input-veld verschijnt",
-//       table: {
-//         defaultValue: { summary: "false" },
-//       },
-//     },
-//   },
-// };
+export default {
+  title: "native-elements/vl-input-field",
+  decorators: [(story) => html`${stylesheet(styles)}${story()}`],
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
```

Comment:
In de docs staat er nu dat je geen stylesheet nodig hebt, terwijl dat wel zo is lijkt me. Ik zie ook geen changes in het build script, waar je componenten moet toevoegen als ze een stylesheet moeten aanbieden.

Reply:
Idd, ik heb ze toegevoegd.